### ISSUE TITLE
Problem with @extend cascaded in class ending with :after

### ISSUE THREAD

**Issue by Cedric-ruiu** [sass/sass#731]
Hi,

I use FireApp v1.11. this last update include SASS v3.2.8 + Compass v0.12.2 installed on Win 7 64 bits.

I have a problem with this code :

```
%foo{ background-color: red; }

.b{
    position: relative;
    .bar{
        @extend %foo;
        &:after{ @extend %foo; }    
    }
}
```

produce :

```
.b .bar { background-color: red; }
.b { position: relative; }
```

instead of :

```
.b .bar, .b .bar:after { background-color: red; }
.b { position: relative; }
```

The last version have a break change, it's a bug or i forgot something?

Cédric R.

**Comment by Cedric-ruiu** [comment]
FireApp v1.10 based on Sass v3.2.7 don't produce this bug.

**Comment by nex3** [comment]
This is fixed by 5bc676942daa24e81f4a473894652a18b8b85a41, which is as yet unreleased.
